codex-ui/dev/Playground.vue

```diff
@@ -1,7 +1,83 @@
 <template>
   <div :class="$style.playground">
     <Heading :level="1">Playground</Heading>
+    <Heading :level="3">Mode:</Heading>
+    <div :class="$style.buttons">
+      <Button
+        size="small"
+        @click="() => setMode('light')"
```

Comment:
in vue it will also work like this

```suggestion
        @click="setMode('light')"
```